guides/common/modules/con_security-content-automation-protocol.adoc

```diff
@@ -7,7 +7,7 @@ SCAP is a framework of several specifications based on XML, such as checklists d
 These specifications are encapsulated as _data stream_ files.
 
 Checklist items in XCCDF, also known as _rules_, express the desired configuration of a system item.
-For example, a rule may specify that no one can log in to a host over SSH using the `root` user account.
+For example, a rule may specify that no one can log in to a host over SSH by using the `root` user account.
```

Comment:
```suggestion
For example, a rule may specify that no one can log in to a host over SSH by using the root user account.
```
[RHSSG](https://redhat-documentation.github.io/supplementary-style-guide/#commands-with-root-privileges): No markup of user names like root.

Reply:
"root privileges" are differnet to me than "the `root` user ...". I lean towards keeping the emphasis here.

Reply:
I'm keeping it marked. The RH SSG section doesn't forbid me from using the markup for user names.